install.sh

```diff
@@ -171,7 +171,10 @@ while getopts "hsinaf" opt; do
 			;;
 	esac
 done
-shift $(("$OPTIND" - 1))
+
+# If we quote here, this will not work on older versions of bash
+# shellcheck disable=2004
+shift $(($OPTIND - 1))
```

Comment:
I think [shellcheck SC2004](https://github.com/koalaman/shellcheck/wiki/SC2004) wants:
```
shift $((OPTIND - 1))
```

With this, you should be able to remove the disable statement

Reply:
nice one @davidpfarrell, done :smile: